docs/contents.md

```diff
@@ -0,0 +1,60 @@
+# Content
+
+You can familiarize yourself with how the project functions by watching the [video](https://www.youtube.com/watch?v=d_RFHXyGSVo) and/or reading the [blog](./blog.md).
```

Comment:
Can we add a section here that says 

## Video 
The video can be found here
## Blog
The blog can be found here.